crates/steel-core/src/primitives/bytevectors.rs

```diff
@@ -289,23 +289,24 @@ pub fn list_to_bytes(value: Vec<u8>) -> Result<SteelVal> {
     Ok(SteelVal::ByteVector(SteelByteVector::new(value)))
 }
 
-/// Append two byte vectors into a new bytevector.
+/// Append multiple byte vectors into a new bytevector.
 ///
 /// # Examples
 /// ```scheme
-/// (bytes-append (bytes 0 1 2) (bytes 3 4 5)) ;; => (bytes 0 1 2 3 4 5)
+/// (bytes-append #u8(0 1 2) #u8(3 4 5)) ;; => #u8(#x00 #x01 #x02 #x03 #x04 #x05)
```

Comment:
I suppose all docs could be updated to use literals like this one. I might get around doing it in this PR or a subsequent one.